### PR TITLE
[release] angular v10 beta

### PR DESCRIPTION
**Pull request by wzhudev**

### CODE REVIEW

File: packages/icons-angular/src/package.json [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
   "contributors": [
     "Wendell Hu <wendellhu95@gmail.com>"
   ],
   "dependencies": {
     "@ant-design/colors": "^3.1.0"
-  },
+  ,    "tslib": "^2.0.0"
+},
   "peerDependencies": {
     "@angular/common": "^9.0.0",
```
Comment (vthinkxie) [inline packages/icons-angular/src/package.json]: peerDependencies needs update too

**Review by vthinkxie — CHANGES_REQUESTED** [review]

File: packages/icons-angular/src/package.json [hunk 2]
```diff
@@ -7,13 +7,13 @@
...
+    "tslib": "^2.0.0"
 },
   "peerDependencies": {
-    "@angular/common": "^9.0.0",
-    "@angular/core": "^9.0.0",
-    "@angular/platform-browser": "^9.0.0",
+    "@angular/common": "^10.0.0",
+    "@angular/core": "^10.0.0",
+    "@angular/platform-browser": "^10.0.0",
     "rxjs": "^6.4.0"
```
Comment (vthinkxie) [inline packages/icons-angular/src/package.json:17]: rxjs should also update?

Comment (wzhudev) [reply packages/icons-angular/src/package.json:17]: Updated

**Review by vthinkxie — APPROVED** [review]
lgtm

**Pull request merged**